deployment/ansible/raster-foundry.yml

```diff
@@ -32,3 +32,7 @@
     - name: Set Environment variable for host user (used to namespace jars and other artifacts in development)
       lineinfile: dest=/etc/environment regexp=^RF_HOST_USER line="RF_HOST_USER={{ host_user }}"
       when: host_user is defined
+
+    - name: Set pattern for core dump location
+      sysctl: name="kernel.core_pattern" value="/tmp/core.%t.%e.%p"
```

Comment:
@jamesmcclain, this appears to be getting created within the context of the container. The core dump file **does not exist** on the Docker host file system in `/tmp`.

Reply:
Sounds positive.  I will report on my progress below.